From 50ace55df604bb0dd799a02f6346f4ea0800acf6 Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Mon, 30 Mar 2020 21:30:04 +0800
Subject: [PATCH 06/22] driver: mxc: vpu_encoder: use the correct format
 placeholder

It is need to use the %lld as the format placeholder when the
variable is a 64bit data.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/vpu_windsor/vpu_encoder_b0.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/mxc/vpu_windsor/vpu_encoder_b0.c b/drivers/mxc/vpu_windsor/vpu_encoder_b0.c
index 44130ca06ba2..df269b3138cb 100644
--- a/drivers/mxc/vpu_windsor/vpu_encoder_b0.c
+++ b/drivers/mxc/vpu_windsor/vpu_encoder_b0.c
@@ -2453,7 +2453,7 @@ static int inc_frame(struct queue_data *queue)
 	list_add_tail(&frame->list, &queue->frame_idle);
 	atomic64_inc(&queue->frame_count);
 
-	vpu_dbg(LVL_DEBUG, "++ frame : %ld\n",
+	vpu_dbg(LVL_DEBUG, "++ frame : %lld\n",
 			atomic64_read(&queue->frame_count));
 
 	return 0;
@@ -2467,7 +2467,7 @@ static void dec_frame(struct vpu_frame_info *frame)
 	if (frame->queue) {
 		atomic64_dec(&frame->queue->frame_count);
 
-		vpu_dbg(LVL_DEBUG, "-- frame : %ld\n",
+		vpu_dbg(LVL_DEBUG, "-- frame : %lld\n",
 				atomic64_read(&frame->queue->frame_count));
 	}
 	VPU_SAFE_RELEASE(frame, vfree);
@@ -4024,7 +4024,7 @@ static int show_instance_others(struct vpu_attr *attr, char *buf, u32 size)
 	}
 
 	num += scnprintf(buf + num, size - num,
-			"\ttotal dma size            :%ld\n",
+			"\ttotal dma size            :%lld\n",
 			atomic64_read(&attr->total_dma_size));
 	num += scnprintf(buf + num, size - num,
 			"\ttotal event msg obj count :%ld\n",
@@ -4037,7 +4037,7 @@ static int show_instance_others(struct vpu_attr *attr, char *buf, u32 size)
 	ctx = get_vpu_attr_ctx(attr);
 	if (ctx) {
 		num += scnprintf(buf + num, size - num,
-			"\ttotal frame obj count     :%ld\n",
+			"\ttotal frame obj count     :%lld\n",
 			atomic64_read(&ctx->q_data[V4L2_DST].frame_count));
 
 		if (test_bit(VPU_ENC_STATUS_HANG, &ctx->status))
-- 
2.17.1

